internal/config/auditsink.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+import (
+	"github.com/spf13/viper"
+)
+
+// AuditSinkConfig contains fields, which enable and configure
+// Flipt's various audit sink mechanisms.
+type AuditSinkConfig struct {
+	Sinks    SinksConfig    `json:"sinks,omitempty" mapstructure:"sinks"`
+	Advanced AdvancedConfig `json:"advanced,omitempty" mapstructure:"advanced"`
+}
+
+func (c *AuditSinkConfig) setDefaults(v *viper.Viper) {
+	v.SetDefault("audit", map[string]any{
+		"sinks": map[string]any{
+			"logFile": map[string]any{
+				"enabled":  "false",
+				"filePath": "./path/to/log/file",
+			},
+		},
+		"advanced": map[string]any{
+			"bufferSize": 2,
+		},
```

Comment:
Just a personal preference / suggestion here. I think it might be better to just keep configuration very explicit about what it controls. Perhaps a `buffer` section with a single item `size` or `capacity` would be more clear. Leaving out the ambiguity of what `advanced` means.

```suggestion
		"buffer": map[string]any{
			"capacity": 2,
		},
```

Reply:
@GeorgeMac Agreed. I think being more explicit is necessary here. My thinking of using the `Advanced` was to have the sentiment that the user should be careful while messing with this configuration. More buffering means more possibility of losing audit events when the Flipt instance crashes (I guess), less durability basically.

Reply:
I see your point r.e. Being cautious here. Maybe that is something we can make clear in the docs. Plus also pick a decent set of defaults.

Maybe we should also add the periodic flush duration in here too.

Reply:
@GeorgeMac Yeah was thinking the same thing. I can add the flush duration here and add some validation on that as well.